JSA-Mawsitsit-Backend/routes/registerRouter.js

```diff
@@ -0,0 +1,55 @@
+const express = require('express');
+const router = express.Router();
+
+const passwordStrength = (password) => {
+  const weakRegex = new RegExp('^((?=.[a-z])|(?=.[0-9])|(?=.*[A-Z]))(?=.{8,})');
+  if (weakRegex.test(password)) {
```

Comment:
lets simplify this, if this returns false, then password has failed
```suggestion
  return weakRegex.test(password)
```